content/en/armory-enterprise/armory-admin/observe/integrations-logging.md

```diff
@@ -0,0 +1,105 @@
+---
+
+## Metrics and logging overview
+
+Metrics allow you to investigate the health of your Armory Enterprise system. Armory's [Observability Plugin]({{< ref "observability-configure.md" >}}) provides a good overview of data over time about what is happening in particular services. You can monitor data points such as overall load, number of errors, and connection speeds.
+
+Logging, on the other hand, is more useful to trace data about what individual accounts and functions are doing within Armory Enterprise. This data provides an overview on applications and errors and is primarily for auditing purposes. Armory Enterprise captures logging data in Echo, so you can configure Echo to send logging data to your data aggregator.
+
+## {{% heading "prereq" %}}
+
+You have set up a data aggregator with a unique endpoint. You need the endpoint and an authorization token when you enable logging in Armory Enterprise. Documentation for some popular data aggregators:
+
+* [Create an HTTP Event Collector in Splunk](https://docs.splunk.com/Documentation/Splunk/8.2.4/Data/UsetheHTTPEventCollector)
+* [Datadog Log Collection](https://docs.datadoghq.com/logs/log_collection/?tab=tcp)
+* [Cloudwatch REST API Logging](https://docs.aws.amazon.com/apigateway/latest/developerguide/set-up-logging.html)
+
+## Enable logging data
+
+The following configuration example shows how to configure Echo to send data to Splunk. If you are using another data aggregator, replace the `url` value with the endpoint for your aggregator.
+
+{{< tabs name="enable-logging" >}}
+{{< tabbody name="Operator" >}}
+
+Add the <b>rest</b> block to the <b>spec.spinnakerConfig.profiles.echo</b> section of your Operator manifest.
+
+{{< prism lang="yaml" >}}
+apiVersion: spinnaker.armory.io/v1alpha2
+kind: SpinnakerService
+metadata:
+  name: spinnaker
+spec:
+  spinnakerConfig:
+    profiles:
+      echo:
+        rest:
+          enabled: true
+          endpoints:
+          - wrap: true
+            url: "http://<HOSTNAME>:<PORT>/services/collector/event?"
+            headers:
+              Authorization: "<TOKEN>"
+            template: '{"event":{{event}} }'
+          insecure: true
+{{< /prism  >}}
+
+{{< /tabbody >}}
+{{< tabbody name="Halyard" >}}
+
+Add the <b>rest</b> block to the <b>echo-local.yml</b> section in your <b>~/.hal/default/profiles/</b> directory.
+
+{{< prism lang="yaml" >}}
+rest:
+  enabled: true
+  endpoints:
+  - wrap: true
+    url: "http://<HOSTNAME>:<PORT>/services/collector/event?"
+    headers:
+      Authorization: "<TOKEN>"
+    template: '{"event":{{event}} }'
+  insecure: true
+{{< /prism  >}}
+
+{{< /tabbody >}}
+{{< /tabs >}}
+
+The `HOSTNAME` and `PORT` should be the same as the TCP listener port available in your aggregator. `TOKEN` is the authorization token for your aggregator.
```

Comment:
The default port for HEC is either 8088 or 8089 depending on your Splunk version. This port will be used if you DO NOT configure a TCP listener. Any TCP listener config is a custom port assignment.

Reply:
With regards to Chris' findings, I placed the following KB article about the ports, and it can be linked to the article: https://support.armory.io/support?id=kb_article_view&sysparm_article=KB0010554